src/test/java/serviceimpl/FileWriterImplTest.java

```diff
@@ -0,0 +1,55 @@
+package serviceimpl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.FileWriter;
+
+public class FileWriterImplTest {
+    private static FileWriter fileWriter;
+    private static final String WAY_TO_REPORT = "src/main/resources/CloseDay.csv";
+    private static final String WRONG_WAY_TO_WRITE = "src/main/resources/CloseDay.csv";
+    private static String expectedString;
+
+    @BeforeClass
+    public static void beforeClass() {
+        expectedString = "fruits, quantity" + System.lineSeparator()
+                + "banana" + "," + "152" + System.lineSeparator()
+                + "apple" + "," + "90";
+    }
+
+    @Test
+    public void fileWriter_Ok() {
+        String expected = expectedString;
+        String actual = getReadFile(WAY_TO_REPORT);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void wrongPath_NotOk() {
```

Comment:
```suggestion
    public void write_wrongPath_NotOk() {
```
change names accordingly